benchmarks/distributed/rpc/ps/metrics/MetricsLogger.py

```diff
@@ -0,0 +1,92 @@
+from .CPUMetric import CPUMetric
+from .CUDAMetric import CUDAMetric
+
+
+class MetricsLogger:
+
+    CUDA = "cuda"
+    CPU = "cpu"
+
+    def __init__(self, rank, metric_class, overwrite_metrics=False):
+        self.rank = rank
+        self.metric_class = metric_class.lower()
+        self.overwrite_metrics = overwrite_metrics
+        self.metrics = {}
+
+    def add_metric(self, metric_type, key, metric_name):
```

Comment:
the API name already contains a metric, we can simplify the args to type, key, name

Reply:
okay.